newrelic/structures_newrelic_one_dashboard.go

```diff
@@ -86,6 +86,13 @@ func expandDashboardVariablesInput(variables []interface{}) []dashboards.Dashboa
 			variable.Type = dashboards.DashboardVariableType(strings.ToUpper(ty.(string)))
 		}
 
+		if i, ok := v["options"]; ok {
+			ignoreTimeRange, is := i.(bool)
+			if is {
+				variable.Options = dashboards.DashboardVariableOptionsInput{IgnoreTimeRange: ignoreTimeRange}
+			}
+		}
+
```

Comment:
@Avinash-Sanpala this section might need correction. It looks like "i" is taken from "options" as `ignoreTimeRange`, however, we need to access the attribute "ignore_time_range" from the Terraform configuration. 

An example of what I mean to describe above is in the function `expandDashboardVariablesInput` for the attribute `nrql_query` in `variables` 

```go
		if q, ok := v["nrql_query"]; ok && len(q.([]interface{})) > 0 {
			variable.NRQLQuery = expandVariableNRQLQuery(q.([]interface{}))
		}
```

Can you please test setting `options` and `ignore_time_range` in the Terraform Configuration via an API debugging tool (this might give an idea on how `ignoreTimeRange` is being set with these changes)